docs/src/openapi-v1.yaml

```diff
@@ -962,6 +962,114 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /pallets/{palletId}/errors:
+    get:
+      tags:
+      - pallets
+      summary: Get a list of errors for a pallet.
+      description: Returns a list of error item metadata for error items of the
+        specified palletId.
+      parameters:
+      - name: palletId
+        in: path
+        description: 'Name or index of the pallet to read error metadata for. Note: the pallet name must match what is
+          specified in the runtime metadata.'
+        required: true
+        schema:
+          type: string
+      - name: onlyIds
+        in: query
+        description: Only return the names (IDs) of the error items instead of all of each error
+          item's metadata.
+        required: false
+        schema:
+          type: boolean
+      - name: at
+        in: query
+        description: Block at which to retrieve a list of
+          the pallet's error items.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                description: Pallet info and Array of errorItemIds.
+                items:
+                  $ref: '#/components/schemas/PalletErrors'
+        "400":
+          description: invalid blockId supplied for at query param
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "404":
+          description: could not find pallet with palletId
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /pallets/{palletId}/errors/{errorItemId}:
+    get:
+      tags:
+      - pallets
+      summary: Get the value of an error item.
+      description: Returns the value stored under the errorItemId.
+      parameters:
+      - name: palletId
+        in: path
+        description: 'Name or index of the pallet to read error metadata for. Note: the pallet name must match what is
```

Comment:
Same thing with the quotes.

Reply:
believe the quotes are for formatting due to the colon in the text. Should these be removed in your opinion?

Reply:
Ahh, cool I didnt notice that. The Quotes can stay!